rust/extractor/src/translate/base.rs

```diff
@@ -559,4 +558,18 @@ impl<'a> Translator<'a> {
             Some(())
         })();
     }
+
+    pub(crate) fn should_be_excluded(&self, item: &impl ast::HasAttrs) -> bool {
+        let Some(sema) = self.semantics else {
+            return false;
+        };
+        for attr in item.attrs() {
+            if let Some((name, tokens)) = attr.as_simple_call() {
+                if name == "cfg" && sema.check_cfg_attr(&tokens) == Some(false) {
+                    return true;
+                }
+            }
+        }
+        false
```

Comment:
Perhaps this would be nicer with `any` and `is_some_and`?
```suggestion
        item.attrs().any(|attr| {
            attr.as_simple_call().is_some_and(|(name, tokens)| {
                name == "cfg" && sema.check_cfg_attr(&tokens) == Some(false)
            })
        })
```